restli-common/src/main/java/com/linkedin/restli/common/validation/RestLiDataSchemaDataValidator.java

```diff
@@ -66,6 +71,24 @@ public ValidationResult validateOutput(RecordTemplate dataTemplate)
     return super.validateOutputAgainstSchema(dataTemplate, _validatingSchema);
   }
 
+  /**
+   * Validator to use to validate the output.
+   * The validator is instantiated in the constructor, so directly returns that if input is equal to _validatingSchema.
+   * @param validatingSchema schema to validate against
+   * @return validator
+   */
+  @Override
+  protected Validator getValidatorForOutput(DataSchema validatingSchema) {
```

Comment:
nit: I personally think "getValidatorForOutput" an incomplete name, it makes more sense to me to say for example "getValidatorForOutputEntityValidation"

Reply:
sounds good, will update

Reply:
updated